projects/ngx-meta/src/core/src/metadata-json-resolver.ts

```diff
@@ -3,15 +3,10 @@ import { InjectionToken } from '@angular/core'
 import { MetadataResolverOptions } from './ngx-meta-metadata-manager'
 import { isObject } from './is-object'
 
-export type MetadataJsonResolver = (
-  values: MetadataValues | undefined,
-  resolverOptions: MetadataResolverOptions,
-) => unknown
 export const METADATA_JSON_RESOLVER = new InjectionToken<MetadataJsonResolver>(
   ngDevMode ? 'NgxMeta JSON Resolver' : 'NgxMetaJR',
   {
-    providedIn: 'root',
```

Comment:
Type is automatically provided. Default is provided in root, so no need to specify it here